redisgears_core/src/lib.rs

```diff
@@ -1376,7 +1441,7 @@ fn function_debug_command(
         }
         _ => (),
     }
-    let backend = get_backend(ctx, backend_name).map_or(
+    let backend = get_backend(backend_name).map_or(
```

Comment:
Same comment here about the ctx

Reply:
Isn't it just makes a simple thing for complicated? And for what reason if eventually the right solution is to wrap the globals which are GIL protected with `RedisGILGuard`?